docs/ota-client-guide/modules/ROOT/pages/troubleshooting-bsp-integration.adoc

```diff
@@ -29,10 +29,10 @@ INITRAMFS_IMAGE = "initramfs-ostree-image"
 
 Building an integration with a new board involves dealing with several different systems. See the following links for more information on this:
 
-* https://www.yoctoproject.org/docs/{yocto-version}/mega-manual/mega-manual.html[Yocto Mega Manual, v2.6]: The Yocto Mega Manual is a concatenation of all the various other reference manuals; it’s usually better to use the individual manuals if you know what you’re looking for. In particular, these three are the most frequently used in the BSP development domain:
-** https://www.yoctoproject.org/docs/{yocto-version}/ref-manual/ref-manual.html[Yocto Reference Manual, v2.6]
-** https://www.yoctoproject.org/docs/{yocto-version}/bsp-guide/bsp-guide.html[Yocto BSP Developer's Guide, v2.6]
-** https://www.yoctoproject.org/docs/{yocto-version}/bitbake-user-manual/bitbake-user-manual.html[Bitbake User Manual, v2.6]
+* https://www.yoctoproject.org/docs/{yocto-version}/mega-manual/mega-manual.html[Yocto Mega Manual, v{yocto-version}]: The Yocto Mega Manual is a concatenation of all the various other reference manuals; it’s usually better to use the individual manuals if you know what you’re looking for. In particular, these three are the most frequently used in the BSP development domain:
+** https://www.yoctoproject.org/docs/{yocto-version}/ref-manual/ref-manual.html[Yocto Reference Manual, v{yocto-version}]
+** https://www.yoctoproject.org/docs/{yocto-version}/bsp-guide/bsp-guide.html[Yocto BSP Developer's Guide, v{yocto-version}]
+** https://www.yoctoproject.org/docs/{yocto-version}/bitbake-user-manual/bitbake-user-manual.html[Bitbake User Manual, v{yocto-version}]

```

Comment:
@tkfu or @ammaboateng do you know if this will work with Antora? Specifically the bit in the link name; the URL part I'm confident in since it works elsewhere. Pity that github doesn't understand it, though.

Reply:
yes, it will work with Antora.